<launch>
    <!-- <node name="voice_control_server" pkg="voice_control" type="voice_control_server2.py" /> -->
</launch>